python/tests/complex_type_tests/test_row_of_arr.py

```diff
@@ -0,0 +1,95 @@
+from tests.aggregate_tests.aggtst_base import TstView, TstTable
+
+
+# Depth 2 tests => Row of Array
+class cmpxtst_row_of_arr_tbl(TstTable):
+    """Define the table used by the ROW of Array tests"""
+
+    def __init__(self):
+        self.sql = """CREATE TABLE row_of_arr_tbl(
+                      id INT,
+                      c1 ROW(i1 INT ARRAY NOT NULL, v1 VARCHAR ARRAY NULL) NOT NULL,
+                      c2 ROW(i2 INT ARRAY NULL, v2 VARCHAR ARRAY NOT NULL))"""
+        self.data = [
+            {"id": 0, "c1": {"i1": [20, 22, 23], "v1": [None]}, "c2": None},
+            {
+                "id": 1,
+                "c1": {
+                    "i1": [12, 13, 77, 10, 2],
+                    "v1": ["hi", "hello", "how are you?"],
+                },
+                "c2": {"i2": [1, 2, None], "v2": ["bye", "see you later!"]},
```

Comment:
please add more nulls - a row with c1 null, a null in array i1, etc. we should make sure we test all combinations

Reply:
there is room for more nulls.
e.g:
`"id": 3, "c1": None, ...`
`"id": 4, "c1": [ "i1": [None, 1]...`
`"id": 4, "c1": [ "i1": None, v1: None`
and more